SDWebImage/SDWebImageDownloaderOperation.m

```diff
@@ -91,36 +97,26 @@ - (void)start {
             }];
         }
 #endif
-
+        NSURLSessionConfiguration *sessionConfig = [NSURLSessionConfiguration defaultSessionConfiguration];
+        sessionConfig.timeoutIntervalForRequest = 15;
+        self.session = [NSURLSession sessionWithConfiguration:sessionConfig
+                                                     delegate:self
+                                                delegateQueue:nil];
```

Comment:
we should add a comment to explain this `nil` as `delegateQueue`. According to the docs: 
> An operation queue for scheduling the delegate calls and completion handlers. The queue need not be a serial queue. If nil, the session creates a serial operation queue for performing all delegate method calls and completion handler calls.